packages/common/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@threefold/common",
+  "version": "1.0.0",
```

Comment:
@0oM4R How to fix that for the upcoming version? just set it and manually link for testing?